test/dataframerow.jl

```diff
@@ -71,8 +90,22 @@ end
     @test copy(view(r, [:c,:b])) == (c = "CC", b = 1.2)
     @test copy(view(r, [:c,:b])) == (c = "CC", b = 1.2)
     @test copy(r[[false, true]]) == (c = "CC",)
+    @test copy(r[r"b"]) == (b = 1.2,)
+    @test copy(view(r, r"b")) == (b = 1.2,)
+    @test copy(view(r, r"b")) == (b = 1.2,)
     r.c = "C"
     @test df.c[3] == "C"
+
+    df = DataFrame([1 2 3 4
+                    5 6 7 8])
+    r = df[1, r"[1-3]"]
+    @test names(r) == [:x1, :x2, :x3]
+    r[:] = 10
+    @test Matrix(df) == [10 10 10 4
```

Comment:
It would be more logical to do this:
```julia
    @test df == DataFrame([10 10 10 4
                           5  6  7 8])
```
Same below.

Reply:
fixed